packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -401,7 +403,8 @@ const handleOriginRequest = async ({
   const { req, res, responsePromise } = lambdaAtEdgeCompat(
     event.Records[0].cf,
     {
-      enableHTTPCompression: manifest.enableHTTPCompression
+      enableHTTPCompression: manifest.enableHTTPCompression,
+      rewrittenUri
```

Comment:
Although you're passing this, it isn't being used by the compat handler. Is that part missing?

Reply:
Good catch, added now.